doc/xmlrpc_client.py

```diff
@@ -0,0 +1,21 @@
+import xmlrpc.client
+
```

Comment:
`doc` folder is inside the module, so it should be `openacademy/doc/***`

Reply:
Currently the `doc` folder is in `openacademy/doc`, isn't it ok?

Reply:
Hi @ejdecena,

I just noticed your module's code is not into a directory "openacademy", it's right at the root of your project (i.e. at the same level as `.gitmodule`).

Odoo modules should be into a separated directory, because a repo could contain several modules.

Could you open a new PR to move all files into a directory, please